### ISSUE TITLE
Alt Attribute

### ISSUE THREAD

**Issue by jim-rosales** [brett099/prj-rev-bwfs-tea-cozy#2]
Great job adding all the images correctly! One thing to consider on future projects is to include the [alt attribute](https://en.wikipedia.org/wiki/Alt_attribute). The alt attribute has a wide array of uses varying from [accessibility](https://www.codecademy.com/articles/accessibility), optimizing your site for search engines, and as a backup, in case your image cannot be rendered. 

For example, you could have done something like the following:
`<img src="resources/images/berryblitz.jpg" alt="Berry Blitz Tea in a mug">`

https://github.com/brett099/prj-rev-bwfs-tea-cozy/blob/6f98415352ace6205bee72710ab07e6c00c5e89c/tea-cozy/index.html#L38
